math_keyboard/pubspec.yaml

```diff
@@ -2,12 +2,12 @@ name: math_keyboard
 description: >-2
   Math expression editing using an on-screen software keyboard or physical keyboard input in a
   typeset input field in Flutter.
-version: 0.2.1
+version: 0.3.0
 homepage: https://github.com/simpleclub/math_keyboard/tree/main/math_keyboard
 
 environment:
-  sdk: '>=2.12.0 <4.0.0'
-  flutter: '>=2.0.0'
+  sdk: '>=3.0.0 <4.0.0'
+  flutter: '>=3.22.2'
```

Comment:
Require latest stable feels weird but everything before depended on `intl 0.18.x`... Wdyt?

Reply:
> but everything before depended on intl 0.18.x

I think in that case we have to go with it 👍